fe/fe-core/src/main/java/com/starrocks/alter/LakeTableSchemaChangeJob.java

```diff
@@ -347,9 +347,8 @@ protected void runPendingJob() throws AlterCancelException {
                                 table.primaryIndexCacheExpireSec(), TTabletType.TABLET_TYPE_LAKE, table.getCompressionType(), 
                                 copiedSortKeyIdxes);
 
-                        Long baseTabletId = partitionIndexTabletMap.row(partitionId).get(shadowIdxId).get(shadowTabletId);
-                        assert baseTabletId != null;
-                        createReplicaTask.setBaseTablet(baseTabletId, 0/*unused*/);
+                        // No need to set base tablet
+                        // createReplicaTask.setBaseTablet(0, 0/*unused*/);
```

Comment:
would rather keep the comment and delete the commented code.

Reply:
Updated